67_RayQueryGeometry/app_resources/common.hlsl

```diff
@@ -0,0 +1,46 @@
+#ifndef RQG_COMMON_HLSL
+#define RQG_COMMON_HLSL
+
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+NBL_CONSTEXPR uint32_t WorkgroupSize = 16;
+
+struct SGeomInfo
+{
+    uint64_t vertexBufferAddress;
+    uint64_t indexBufferAddress;
+
+	uint32_t indexType; // 16 bit, 32 bit or none
+	uint32_t vertexStride;
```

Comment:
pro tip, your struct is already 64 byte aligned, you may wanna keep the other members down to have no padding, so lets say
```
uint32_t indexType : 2;
uint32_t vertexStride : 30;
uint32_t byteOffset;
```
which add up to 64 bits